zulipterminal/model.py

```diff
@@ -305,6 +305,17 @@ def get_messages(self, *,
             return True
         return False
 
+    def get_topics_in_stream(self, stream_list: List[int]) -> None:
```

Comment:
I think this could just be an `Iterable[int]` ?

Reply:
Will do so. But I am curious to know why you are asking me to do the change to `Iterable` from `List`?

Reply:
From what I remember of the method, it was just looking at the elements inside of it; it could have been a tuple (so still a sequence), but the order itself is also not important, which probably makes iterable the more general form.

If the method ends up requiring a list, then we can put it as such (for the sake of mypy & docs), but until it does, this frees up the passing-in code to use whatever it likes.

Reply:
I see you've updated this; let me know if you want to discuss this further.

Reply:
Oops, I had missed this updated. I think my question was justified. Thanks for the explanation :)